subsys/net/l2/openthread/openthread.c

```diff
@@ -279,6 +297,29 @@ enum net_verdict ieee802154_radio_handle_ack(struct net_if *iface,
 	return NET_CONTINUE;
 }
 
+#if defined(CONFIG_OPENTHREAD_JOINER_AUTOSTART)
+void openthread_joiner_autostart(struct net_if *iface)
+{
+	struct openthread_context *ot_context = net_if_l2_data(iface);
+	otError error;
+
+	if (!otDatasetIsCommissioned(ot_context->instance)) {
+		error = otJoinerStart(ot_context->instance,
+					CONFIG_OPENTHREAD_JOINER_PSKD, NULL,
```

Comment:
Sorry for nitpicking but I just noticed that there is some style issue still present.
Please align the parameters to same column.

Reply:
Thanks! I fixed it.